bcc/module.go

```diff
@@ -140,6 +142,11 @@ func (bpf *Module) Close() {
 		C.free(unsafe.Pointer(tpCategoryCS))
 		C.free(unsafe.Pointer(tpNameCS))
 	}
+	for _, v := range bpf.rawTracepoints {
+		C.bpf_close_perf_event_fd((C.int)(v))
```

Comment:
Do we need this?

I wrote a [simple example program](https://github.com/iovisor/gobpf/blob/iaguis/example-raw-tp/examples/bcc/rawtracepoints/rawtracepoints.go) and when I Ctrl-C I see:

```
ioctl(PERF_EVENT_IOC_DISABLE) failed: Inappropriate ioctl for device
```

I see that in [`bpf_attach_raw_tracepoint`](https://github.com/iovisor/bcc/blob/v0.7.0/src/cc/libbpf.c#L1097) there's no `bpf_attach_tracing_event()`, unlike in [`bpf_attach_tracepoint`](https://github.com/iovisor/bcc/blob/v0.7.0/src/cc/libbpf.c#L1084) so I don't think we should do this.

Reply:
Good point, removed.